apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -14,27 +14,35 @@ import {
   Separator,
 } from "@webstudio-is/design-system";
 import { UndoIcon } from "@webstudio-is/icons";
-import { useIsFromCurrentBreakpoint } from "./use-is-from-current-breakpoint";
 import { isFeatureEnabled } from "~/shared/feature-flags";
+import type { StyleInfo } from "./style-info";
 
 type PropertyProps = {
+  currentStyle: StyleInfo;
```

Comment:
we should think of a better name than `currentStyle`, because style info contains all types of sources

Reply:
style?

Reply:
yeah, better